usersync/chooser.go

```diff
@@ -161,3 +170,13 @@ func (c standardChooser) evaluate(bidder string, syncersSeen map[string]struct{}
 
 	return syncer, BidderEvaluation{Bidder: bidder, Status: StatusOK}
 }
+
+func (c standardChooser) requiresCnxSync(bidder string, syncerKey string, cookie *Cookie) bool {
+	if bidder != "connatix" {
+		return false
+	}
+
+	uid, _, _ := cookie.GetUID(syncerKey)
```

Comment:
cookie.GetUID is used in HasLiveSync as well, is this inefficient?

Reply:
`getUID` implementation is predominantly a key lookup in a map: `cookie.uids[key]` so no major performance concerns. Also, this is not meant for prod, just for a testing phase

Reply:
you are also doing an expiry check, just seems you don't know what other code someone might add into GetUID, but I will approve :)

Reply:
you're right, sorry my IDE had the `if` block folded. I'm checking with @ionut2  but now concerned since it seems that we just want this for testing purposes; this is not intended for prod

Reply:
it's not something we could control in prod since the publisher will use its own Prebid Server instance and Player Bidding will use the CNX client side adapter in prod